engine/engine/src/engine.cpp

```diff
@@ -1860,6 +1884,11 @@ namespace dmEngine
 #endif
                 dmJobThread::Update(engine->m_JobThreadContext, jobthread_max_time_us);
 
+                {
+                    DM_PROFILE("Resource");
+                    dmResource::UpdateFactory(engine->m_Factory); // Process Reload event
+                }
```

Comment:
This function only processes the "Reload" event, any jobs it has are handled by the jobthread